node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -63,6 +63,7 @@ trait GossipHandler: NamedType + Send /* Send because lazily-written tests requi
         database: &mut NeighborhoodDatabase,
         agrs: Vec<AccessibleGossipRecord>,
         gossip_source: SocketAddr,
+        peer_addrs: &[IpAddr],
```

Comment:
Can you think of a better name than `peer_addrs`? When you're thinking about Gossip, you're probably not thinking about connection progress messages, and it's difficult to understand what a peer might be in this context. There's `cpm_recipient` right below it; maybe these two parameters could be named similarly to make their association clear?

Reply:
I used the name `connection_progress_peers`. I think it's conveying the correct idea.